### PR TITLE
Navigate between voltage level in diagram :

### PR DESCRIPTION
**Pull request by FranckLecuyer**
New classes FeederBranchNode and FeederLineNode
Change inheritance for Feeder2WTNode
Add new interface DisplayVoltageLevel, implemented by SubstationDiagramViewer
Add the nearby voltageLevel id in NodeMetadata and NodeHandler
Add a tooltip on the tabs, with the substation/voltageLevel name
Display id or name of the voltageLabel in diagram
Implement the display of the nearby voltageLevel, when clicking on a line or a transformer in the diagram

Signed-off-by: Franck LECUYER <franck.lecuyer@rte-france.com>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Allowing the navigation between voltage levels in the diagram, by clicking on line or transformer nodes

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

**Comment by FranckLecuyer** [comment]
Sonar code analysis failed, but the coverage on new code is over 80% in substation-diagram-core

File: substation-diagram/substation-diagram-core/src/test/resources/substDiag.svg [hunk 1]
```diff
@@ -75,20 +75,21 @@
...
             <text x="-5.0" font-size="8" y="-5.0" class="component-label" font-family="Verdana" transform="rotate(0,0,0)">bbs1</text>
         </g>
-        <g class="substation-diagram BUSBAR_SECTION bbs2" id="bbs2" transform="translate(580.0,360.0)">
+        <g class="substation-diagram BUSBAR_SECTION bbs2" id="bbs2" transform="translate(630.0,360.0)">
             <line y2="0" x1="0" class="bus_vl1" x2="280.0" y1="0"/>
             <text x="-5.0" font-size="8" y="-5.0" class="component-label" font-family="Verdana" transform="rotate(0,0,0)">bbs2</text>
         </g>
         <g class="substation-diagram BUSBAR_SECTION bbs3" id="bbs3" transform="translate(80.0,385.0)">
-            <line y2="0" x1="0" class="bus_vl1" x2="430.0" y1="0"/>
+            <line y2="0" x1="0" class="bus_vl1" x2="480.0" y1="0"/>
```
Comment (geofjamg) [inline substation-diagram/substation-diagram-core/src/test/resources/substDiag.svg]: How do you explain this change in the rendering?

File: substation-diagram/substation-diagram-core/src/test/resources/TestCase11SubstationGraphHorizontal.svg [hunk 2]
```diff
@@ -27,7 +27,7 @@
...
...E_ONE_UP .arrow-down { visibility: hidden;}.ARROW1_trf8_THREE_ONE_DOWN .arrow-down {stroke: black; fill: black; fill-opacity:1;  visibility: visible;}.ARROW1_trf8_THREE_ONE_DOWN .arrow-up { visibility: hidden;}.ARROW2_trf8_THREE_ONE_UP .arrow-up {stroke: blue; fill: blue; fill-opacity:1; visibility: visible;}.ARROW2_trf8_THREE_ONE_UP .arrow-down { visibility: hidden;}.ARROW2_trf8_THREE_ONE_DOWN .arrow-down {stroke: blue; fill: blue; fill-opacity:1;  visibility: visible;}.ARROW2_trf8_THREE_ONE_DOWN .arrow-up { visibility: hidden;}.ARROW1_trf8_THREE_TWO_UP .arrow-up {stroke: black; fill: black; fill-opacity:1; visibility: visible;}.ARROW1_trf8_THREE_TWO_UP .arrow-down { visibility: hidden;}.ARROW1_trf8_THREE_TWO_DOWN .arrow-down {stroke: black; fill: black; fill-opacity:1;  visibility: visible;}.ARROW1_trf8_THREE_TWO_DOWN .arrow-up { visibility: hidden;}.ARROW2_trf8_THREE_TWO_UP .arrow-up {stroke: blue; fill: blue; fill-opacity:1; visibility: visible;}.ARROW2_trf8_THREE_TWO_UP .arrow-down { visibility: hidden;}.ARROW2_trf8_THREE_TWO_DOWN .arrow-down {stroke: blue; fill: blue; fill-opacity:1;  visibility: visible;}.ARROW2_trf8_THREE_TWO_DOWN .arrow-up { visibility: hidden;}]]></style>
```
Comment (geofjamg) [inline substation-diagram/substation-diagram-core/src/test/resources/TestCase11SubstationGraphHorizontal.svg]: The diff is hard to understand, what kind of change has  been done in the css? How is it linked to substation navigation?

File: substation-diagram/substation-diagram-core/src/test/resources/substDiag.svg (hunk 1 above)
Comment (FranckLecuyer) [reply substation-diagram/substation-diagram-core/src/test/resources/substDiag.svg]: In order to improve the lines and conditions coverage, I enriched some tests (TestCase11SubstationGraph, TestCase12GraphWith3WT), which explains the difference in the generated SVG test files.

File: substation-diagram/substation-diagram-core/src/test/resources/TestCase11SubstationGraphHorizontal.svg (hunk 2 above)
Comment (FranckLecuyer) [reply substation-diagram/substation-diagram-core/src/test/resources/TestCase11SubstationGraphHorizontal.svg]: Same explanation as above

**Pull request merged**